package.json

```diff
@@ -98,7 +98,7 @@
     "mocha": "^10.2.0",
     "mocha-chai-jest-snapshot": "^1.1.4",
     "nyc": "^15.1.0",
-    "prettier": "^2.8.8",
+    "prettier": "^3.0.0",
     "prettier-plugin-pkg": "^0.18.0",
     "prettier-plugin-svelte": "^2.10.1",
```

Comment:
https://github.com/sveltejs/prettier-plugin-svelte/issues/317

Reply:
Thanks for letting me know 😄